api/apps/geoprocessing/src/modules/scenario-planning-units-inclusion/scenario-planning-units-inclusion-processor.ts

```diff
@@ -1,12 +1,108 @@
 import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Brackets, In, Repository, WhereExpression } from 'typeorm';
+import { MultiPolygon, Polygon } from 'geojson';
+import { flatten } from 'lodash';
 import { Job } from 'bullmq';
+
 import { WorkerProcessor } from '@marxan-geoprocessing/modules/worker';
 import { JobInput } from '@marxan-jobs/planning-unit-geometry';
+import {
+  LockStatus,
+  ScenariosPlanningUnitGeoEntity,
+} from '@marxan/scenarios-planning-unit';
 
 @Injectable()
 export class ScenarioPlanningUnitsInclusionProcessor
   implements WorkerProcessor<JobInput, true> {
-  async process(_job: Job<JobInput, true>): Promise<true> {
+  constructor(
+    @InjectRepository(ScenariosPlanningUnitGeoEntity)
+    private readonly scenarioPlanningUnitsRepo: Repository<ScenariosPlanningUnitGeoEntity>,
+  ) {}
+
+  async process(job: Job<JobInput, true>): Promise<true> {
+    const scenarioId = job.data.scenarioId;
+    const includeGeo = job.data.include?.geo;
+    const excludeGeo = job.data.exclude?.geo;
+
+    const geometriesIdsToInclude: string[] = [];
+    const geometriesIdsToExclude: string[] = [];
+
+    if (includeGeo) {
+      const targetGeometries = flatten(
+        includeGeo.map((collection) => collection.features),
+      ).map((feature) => feature.geometry);
```

Comment:
`flatten` + `map` = `flatMap`

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/258/commits/67de80971e63759ffa982c2faa81a3707c79d160

Reply:
fixed fix in https://github.com/Vizzuality/marxan-cloud/pull/258/commits/0a7bc2124391d1ed444adb2a5d97fd765bdcf852